src/main.rs

```diff
@@ -1,3 +1,6 @@
+#![allow(dead_code)]
```

Comment:
Is there a way to explicitly place this before dead code?

Reply:
Sure